Failing CLI test due to click update from 8.1.8 to 8.2.0

[This CI job](https://github.com/DiamondLightSource/httomo/actions/runs/15025156518/job/42223971921?pr=583) shows the following test failure:
```
FAILED tests/test_cli.py::test_cli_noargs_shows_help - assert 2 == 0
 +  where 2 = <Result SystemExit(2)>.exit_code
```

This is due to click v8.2.0 being released recently (10/05/25) which changed the status code for when the CLI is invoked without any args: https://github.com/pallets/click/blob/8.2.0/CHANGES.rst?plain=1#L59-L61

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -24,8 +24,8 @@ def test_cli_help_shows_help():
 def test_cli_noargs_shows_help():
     runner = CliRunner()
     result = runner.invoke(main)
-    assert result.exit_code == 0
-    assert result.stdout.strip().startswith("Usage: ")
+    assert result.exit_code == 2
+    assert result.stderr.strip().startswith("Usage: ")
 
 
 def test_cli_check_pass_data_file(standard_loader, standard_data):

